data/moves.js

```diff
@@ -4883,15 +4863,8 @@ exports.BattleMovedex = {
 		pp: 25,
 		priority: 0,
 		flags: {contact: 1, protect: 1, mirror: 1},
-		onHit: function (target, pokemon) {
-			pokemon.addVolatile('fellstinger');
-		},
-		effect: {
-			duration: 1,
-			onAfterMoveSecondarySelf: function (pokemon, target, move) {
-				if (!target || target.fainted || target.hp <= 0) this.boost({atk:3}, pokemon, pokemon, move);
-				pokemon.removeVolatile('fellstinger');
-			},
+		onAfterMoveSecondarySelf: function (pokemon, target, move) {
```

Comment:
Since you're refactoring this anyway, could you make Fell Stinger work regardless of Sheer Force? No idea why it was implemented this way.

Reply:
Not easily no, since Sheer Force removes the only events the Move itself receives after it is used. (And if you try to do it after it is Run then you run into problems with moves that call it.)